pyhook/CMakeLists.txt

```diff
@@ -14,6 +14,9 @@ endif()
 target_link_libraries(fontforge_pyhook PRIVATE fontforge)
 target_link_libraries(psMat_pyhook PRIVATE fontforge)
 
+target_link_libraries(fontforge_pyhook PRIVATE Python3::Module)
```

Comment:
It looks like this is only available in CMake 3.15 or later, so we'd need to update the backport accordingly

Reply:
Thanks for taking care of that!